### PR TITLE
Bh/submission activity item

### PR DESCRIPTION
**Pull request by howebenhowe**
Implements `submission-created`, `submission-deleted`, and `submission-status-changed` activity items.

TODO: include submission activity items in the activity feed, but combine submission creation and pub creation as they are the same thing from the user's perspective.

### CODE REVIEW

**Comment by howebenhowe** [comment]
@idreyn Mr. `ActivityItem`, how does a person work on the renderers and title functions for an activity item when we don't have any `submission` activity in the DB yet? Got anything slicker and more reflective of user generated data than one-off insert statements into the DB?

File: stubstub/activity.ts [hunk 1]
```diff
@@ -29,16 +28,26 @@ export const expectCreatedActivityItem = <Result extends Record<string, any>>(
...
+		toMatchObject: async (values) => {
+			const result = await promise;
+			await finishDeferredTasks();
+			const { query, assertion } = splitQueryFromAssertion(values);
+			const item = await ActivityItem.findOne({
+				where: { ...query, createdAt: { [Op.gte]: cutoffTime } },
+			});
+			expect(item).toMatchObject(assertion);
+			return result;
+		},
```
Comment (3mcd) [inline stubstub/activity.ts:51]: Was splitting this function in two a matter of preference (e.g. to keep `toMatchObject` closer to Jest's version), or was it necessary for some other reason?

File: client/containers/DashboardOverview/helpers/QuickActions.tsx [hunk 2]
```diff
@@ -6,7 +6,7 @@ require('./quickActions.scss');
 
 export type QuickAction = {
 	icon: IconName;
-	label: React.ReactNode;
+	label: string;
```
Comment (3mcd) [inline client/containers/DashboardOverview/helpers/QuickActions.tsx:9]: Super minor (feel free to ignore): at a quick glance it looks like we use string-only values for quick action labels. But I wonder if we could preserve accepting an arbitrary React.Node and instead use the `href` as the key?

**Review by 3mcd — APPROVED** [review]
LGTM! But I'd still implore @idreyn to take a quick glance since he's 👑 of activity items.

**Pull request merged**